### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ module.exports = {
   env: {
     browser: true
   },
-  rules: {},
+  rules: {
+    "ember/no-new-mixins": "warn"
```
Comment (anehx) [inline .eslintrc.js]: Isn't there a new rule "no-jquery" which is in the default set?

File: package.json [hunk 2]
```diff
@@ -35,49 +34,48 @@
...
+    "eslint-plugin-ember": "7.1.0",
+    "eslint-plugin-node": "^10.0.0",
+    "eslint-plugin-prettier": "3.1.1",
+    "husky": "^3.0.8",
+    "lint-staged": "^9.4.1",
     "loader.js": "^4.7.0",
-    "prettier": "^1.13.7",
-    "qunit-dom": "^0.8.4"
+    "prettier": "^1.18.2",
+    "qunit-dom": "^0.9.0"
```
Comment (anehx) [inline package.json]: We should pin the dev dependencies to an exact version

File: .eslintrc.js (hunk 1 above)
Comment (Yelinz) [reply .eslintrc.js]: No, that is in 3.13. But I added it anyways.

File: package.json (hunk 2 above)
Comment (Yelinz) [reply package.json]: Done

File: .eslintrc.js (hunk 1 above)
Comment (anehx) [reply .eslintrc.js]: Why aren't you upgrading to 3.13?

Comment (Yelinz) [reply .eslintrc.js]: 3.12 is the LTS

Comment (Yelinz) [reply .eslintrc.js]: Updated to latest

**Pull request merged**